tests/loggers/test_all.py

```diff
@@ -343,6 +343,47 @@ def _test_logger_created_on_rank_zero_only(tmpdir, logger_class):
     assert trainer.state.finished, f"Training failed with {trainer.state}"
 
 
+class AllRankLogger(LightningLoggerBase):
```

Comment:
https://github.com/PyTorchLightning/pytorch-lightning/blob/master/tests/trainer/logging_/test_distributed_logging.py - the PR needs to update these tests. we could directly update them and remove the changes from here to confirm distributed logging works as expected.